fingerprint2.js

```diff
@@ -359,13 +359,13 @@
         }
         var detect = function (font) {
             var detected = false;
-            for (var index in baseFonts) {
+            baseFonts.forEach(function() {
```

Comment:
Looks like you're right:

http://caniuse.com/#search=forEach

I can change to a C style for loop..